tools/changelogger/src/render.rs

```diff
@@ -58,6 +58,9 @@ pub struct RenderArgs {
     pub source_to_truncate: PathBuf,
     #[clap(long, action)]
     pub changelog_output: PathBuf,
+    /// where should smity_rs entries for server sdk 
+    #[clap(long, action)]
+    pub server_changelog_output: Option<PathBuf>,
```

Comment:
I'm not sure I'm aligned yet on having multiple changelog files. What is your reasoning for this, and if we have more than just client and server in the future, how does this look?

Reply:
At the moment, the server side SDK is missing some features that the clients are specifically waiting for.  Also, at this stage, we would be carrying out more frequent breaking changes than the client SDK is. Having a separate server changelog allows customers to find these changes / features more easily by just looking at the server side document. 

Moving forward, once the server SDK is more stable,  we can deprecate this changelog and start writing to the single changelog file.